src/internal/modules/charge-information/pre-handlers.js

```diff
@@ -63,8 +64,16 @@ const loadIsChargeable = async request => {
 const loadDefaultCharges = async request => {
   const { licenceId } = request.params;
   try {
+    const draftChargeInfo = await loadDraftChargeInformation(request);
+    const startDate = new Date(draftChargeInfo.dateRange.startDate);
+    //  Find non 'draft' CVs where the draft CV start date is in the date range of
+    //  charge versions then pick the one with the greatest version number.
     const versions = await services.water.licences.getLicenceVersions(licenceId);
-    const version = versions.find(v => v.status === 'current');
+    const version = versions.filter(v => {
+      return v.status !== 'draft' && moment.range(v.startDate, v.endDate).contains(startDate);
+    }).reduce((preVal, curVal) => {
+      return (preVal.issue + preVal.increment > curVal.issue + curVal.increment) ? preVal : curVal;
+    });
 
```

Comment:
I'm not certain this will work - e.g. would issue 2 + increment 1 (=3) be given the same sort value as issue 1 + increment 2 (=3)?+

Reply:
One possibility:
```
const getPaddedVersionString = version => version.toString().padStart(9, '0');
const getSortableVersionNumber = (issue, increment) => parseFloat(`${ getPaddedVersionString(issue)}.${ getPaddedVersionString(increment)}`);
```